test/Exercism.Representers.CSharp.IntegrationTests/Solutions/DictionaryInitializations/ComplexObjectInit/expected_representation.txt

```diff
@@ -0,0 +1,10 @@
+public class PLACEHOLDER_1
+{
+    private Dictionary<object, object> PLACEHOLDER_2 = new Dictionary<object, object>{{(Func<Dictionary<object, object>, Dictionary<object, object>>)(PLACEHOLDER_3 =>
+    {
+        PLACEHOLDER_3 = new Dictionary<object, object>{{null, new Dictionary<int, string>{{1, "one"}, {2, "two"}, }}, };
```

Comment:
I kept the comma because it was an indication that the solutions using collection initialization syntax had actually been processed.  In fact, we don't need to do any processing on them.  I felt more comfortable (in control) processing them but I really can't think of a good reason for retaining that approach.

Reply:
Actually mixed nested dictionaries (I look forward to encountering one of those) make excluding collection initialization from the processing tiresome - so no change.